mediasoup-client/src/main/java/org/mediasoup/droid/Device.java

```diff
@@ -152,10 +167,11 @@ public RecvTransport createRecvTransport(
       @NonNull String id,
       @NonNull String iceParameters,
       @NonNull String iceCandidates,
-      @NonNull String dtlsParameters)
+      @NonNull String dtlsParameters,
+      @NonNull String sctpParameters)
```

Comment:
Let's keep API the same as libmediasoup. Add a separate `createSendTransport` with the `sctpParameters` parameter

```c++
		SendTransport* CreateSendTransport(
		  SendTransport::Listener* listener,
		  const std::string& id,
		  const nlohmann::json& iceParameters,
		  const nlohmann::json& iceCandidates,
		  const nlohmann::json& dtlsParameters,
		  const nlohmann::json& sctpParameters,
		  const PeerConnection::Options* peerConnectionOptions = nullptr,
		  const nlohmann::json& appData                        = nlohmann::json::object()) const;
		SendTransport* CreateSendTransport(
		  SendTransport::Listener* listener,
		  const std::string& id,
		  const nlohmann::json& iceParameters,
		  const nlohmann::json& iceCandidates,
		  const nlohmann::json& dtlsParameters,
		  const PeerConnection::Options* peerConnectionOptions = nullptr,
		  const nlohmann::json& appData                        = nlohmann::json::object()) const;
```